src/main/scala/org/allenai/plugins/CoreSettingsPlugin.scala

```diff
@@ -1,14 +1,36 @@
 package org.allenai.plugins
 
-import com.typesafe.sbt.SbtScalariform
-import sbt.Keys._
-import sbt._
-
+import com.typesafe.sbt.{ SbtGit, SbtScalariform }
+import com.typesafe.sbt.git.{ DefaultReadableGit, GitReadonlyInterface, ReadableGit }
+import sbt.{
+  // Implicit conversion to allow `%` on String.
```

Comment:
It's not clear what this comment applies to.  I assume `toGroupID` is a special implicit converter?